llama_stack/core/server/auth.py

```diff
@@ -15,7 +15,7 @@
 from llama_stack.core.server.routes import find_matching_route, initialize_route_impls
 from llama_stack.log import get_logger
 
-logger = get_logger(name=__name__, category="auth")
+logger = get_logger(name=__name__, category="core")
```

Comment:
is this right? or should we instead be adding `auth` as a valid category in `log.py`